crypto/x509/build.info

```diff
@@ -13,5 +13,6 @@ SOURCE[../../libcrypto]=\
         v3_pku.c v3_int.c v3_enum.c v3_sxnet.c v3_cpols.c v3_crld.c v3_purp.c \
         v3_info.c v3_akeya.c v3_pmaps.c v3_pcons.c v3_ncons.c \
         v3_pcia.c v3_pci.c \
+        punycode.c \
```

Comment:
From a layout point of view, I see no reason why this should be in `crypto/x509`.  This should be a set of generic internal utility routines, which I would place in `crypto/`, and have a corresponding header file in `include/internal/` (the contents of which you already have in punycode.c, see the comment "Public interface (would normally go in its own .h file)").

Reply:
Sure. This code may become useful later if we want to print Punycode hostnames in a user-friendly manner (and maybe even accept "native" representations in, e.g., 'req' command). 
What do you think to be a relevant place?

Reply:
> What do you think to be a relevant place?

I think you should read my comment again :wink:

Reply:
Sorry. Got the idea, though I think somewhen it will make sense to move internal utility files to a special utility folder.

Reply:
Moved to crypto/